css/css-fonts/font-size-adjust-013.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>CSS Test: font-size-adjust property</title>
+    <link rel="match" href="font-size-adjust-013-ref.html">
+    <link rel="help" href="https://www.w3.org/TR/css-fonts-5/#font-size-adjust-prop">
+    <style>
+        @font-face {
+            font-family: 'ahem-ex-500';
+            src: url('./resources/ahem-ex-500.otf') format('opentype');
+        }
+        @font-face {
+            font-family: 'ahem-ex-250';
+            src: url('./resources/ahem-ex-250.otf') format('opentype');
+        }
+        .wrapper {
```

Comment:
I am not sure this wrapper adds much - how do you see that?

Reply:
I think the original author intended to make it clear what font was used with explainable indicators.
I removed the wrapper.